### ISSUE TITLE
Requiring this package will break fresh installs in some cases

### ISSUE THREAD

**Issue by Dropa** [zaporylie/composer-drupal-optimizations#7]
If you will create fresh install with [drupal project](https://github.com/drupal-composer/drupal-project.git), then run `composer install`, the `web/` directory will be created with various files, such as `autoload.php` and `index.php`

So, if you do 
```bash
$ git clone https://github.com/drupal-composer/drupal-project.git
$ cd drupal-project
$ composer require zaporylie/composer-drupal-optimizations
$ ls web/
```

it should look like this (yes, there's also hidden files);
```bash
$ ls web/
autoload.php  core  index.php  modules  profiles  robots.txt  sites  themes  update.php  web.config
```

But if we start all over again 
```bash
$ cd .. 
$ rm -rf drupal-project
```
and this time we install some additional package (this does not happen with all, but at least RC-version of name module will do this)

```bash
$ git clone https://github.com/drupal-composer/drupal-project.git
$ cd drupal-project
$ composer require drupal/name zaporylie/composer-drupal-optimizations
$ ls web/
```

now we don't have all the necessary files, this time there is no hidden files either.
```bash
ls web/
core  modules  profiles  sites  themes
```

**Comment by zaporylie** [comment]
Thanks for detailed steps on how to reproduce the issue. For some reason, https://github.com/drupal-composer/drupal-scaffold doesn't trigger.

**Comment by zaporylie** [comment]
This is fixed via #5
